html/interaction/focus/sequential-focus-navigation-and-the-tabindex-attribute/tabindex-getter.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: tabIndex getter return value</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#dom-tabindex">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<input>
+<input type="hidden">
+<button>button</button>
+<button disabled>button</button>
+<button hidden>button</button>
+<a>a</a>
+<a href="#">a</a>
+<textarea></textarea>
+<select></select>
+<iframe width="10" height="10"></iframe>
+<span></span>
+<div></div>
+<div id="hostDelegatesFocus"></div>
+<div id="hostNonDelegatesFocus"></div>
+<script>
+hostDelegatesFocus.attachShadow({ mode: "open", delegatesFocus: true });
```

Comment:
I'd prefer if we used `document.getElementById` or some such to get the element, but I guess we have to support this functionality forever anyway...

Reply:
I think using querySelector is nicer here because we don't need to have IDs + the selectors are already descriptive, so I'm keeping it if that's OK.

Reply:
querySelector is fine with me too, but currently you're using neither, right? You're relying on the ID being exposed as a property on the global object.

Reply:
Oh oops totally didn't check the line. Thank you and sorry, changed it to use getElementById! :D